eventstore/model.go

```diff
@@ -32,7 +34,7 @@ var (
 // Model contains instances of a schema, and provides operations
 // for creating, updating, deleting, and quering them.
 type Model struct {
-	schema       *jsonschema.Schema
+	name         string
```

Comment:
Now instead of using the json schema `Ref` as the identifier of the model, we use the `name` that the user chose. Seems to simplify things and makes more sense since the client chose that name for a reason.